settings.gradle

```diff
@@ -40,11 +40,14 @@ includeWithFlags ':it:server',                'java', 'relocate'
 includeWithFlags ':it:spring:boot-tomcat',    'java', 'relocate'
 includeWithFlags ':it:spring:boot-tomcat8.5', 'java', 'relocate'
 includeWithFlags ':testing-internal',         'java', 'relocate'
-includeWithFlags ':thrift0.11',               'java', 'relocate'
+includeWithFlags ':thrift0.12',               'java', 'relocate'
 
 // Unpublished non-Java projects
 includeWithFlags ':docs-client'
 
+// Unpublished dependency management project to create a virtual platform.
+includeWithFlags ':dependencies', 'dependencies'
```

Comment:
Perhaps we could make this done in `settings-flags.gradle` and use the name and flag that's less likely to clash with user modules, e.g. `dependency-management`?

Reply:
Good idea!